pygeo/constraints/DVCon.py

```diff
@@ -118,7 +118,7 @@ def setSurface(self, surf, name="default", addToDVGeo=False, DVGeoName="default"
         if name in self.surfaces.keys():
             raise KeyError("Surface names must be unique. Repeated surface name: " + str(name))
 
-        self.surfaces[name] = list()
+        self.surfaces[name] = []
```

Comment:
is `list()` too old fashioned?

Reply:
It feels kinda unnecessary, like using `x = str()` instead of `x = ""`.